app/views/layouts/_seo.html.erb

```diff
@@ -0,0 +1,50 @@
+<%# This belongs inside the <head> tag %>
+
+<link rel="publisher" href="https://plus.google.com/u/0/+IndentLabs" />
+
+<%=
+  # Make sure description is no longer than 155 characters
+  site_name = "Notebook"
+  site_description = "Notebook is a set of tools for writers, game designers, and roleplayers to create magnificent universes, and everything within them."
+  site_url = request.host
+  site_image = "http://www.notebook.ai/assets/card-headers/hero-d5161eb41a02535f6656af83cacbdb8c.jpg"
+
+  display_meta_tags description: site_description,
+  # Recommended keywords tag length: up to 255 characters, 20 words.
+  keywords: %w[writing author fiction character universe location nanowrimo],
+  canonical: site_url,
+  twitter: {
+    card: "summary_large_image",
+    title: site_name,
```

Comment:
Will this always put "Notebook" as the Twitter link title? Might be best to inherit page title if we can, since it'll always have Notebook on the end, but also have content names prepended if it's a link to some content.

For example, the title for http://www.notebook.ai/plan/characters/37 is "Tristan Elias - Notebook".